aapcs64/aapcs64.rst

```diff
@@ -716,6 +805,40 @@ The FPCR is used to control the behavior of the floating-point unit. It is a glo
 
 - All other bits are reserved and must not be modified. It is not defined whether the bits read as zero or one, or whether they are preserved across a public interface.
 
+Scalable Vector Registers
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Arm 64-bit architecture also defines an optional set of thirty-two
+scalable vector registers, z0-z31. Each register extends the
+corresponding SIMD and Floating-Point register so that it can hold the
+contents of a single Scalable Vector Type (see `Scalable Vectors`_).
+That is, scalable vector register z0 is an extension of SIMD and
+Floating-Point register v0.
+
+z0-z7 are used to pass scalable vector arguments to a subroutine, and to
+return scalable vector results from a function. If a subroutine takes
+arguments in scalable vector or predicate registers, or if it is a
```

Comment:
I'm tripping over this wording. I'd just be verbose about  it:

```suggestion
arguments in scalable vector registers or predicate registers, or if it is a
```

Reply:
Done.